navigator-menu/src/components/project-menu-components/Home-landing.styles.jsx

```diff
@@ -0,0 +1,33 @@
+import tw from 'tailwind-styled-components';
+
+export const HomeHeadingWrapper = tw.div`
+    w-full 
+    text-center 
+    py-[12rem]
+`;
+
+export const H1 = tw.h1`
```

Comment:
This is not a task, just a comment: sometimes having a single use component that fits on one line and is easy to read through is beneficial to be left in the DOM. It's up to you. But if you're going to create a component - name it properly. For example, instead of `H1` I'd name this `Heading1White` or something similar.

Again, this is a style difference, so I am not requiring a change here.